src/renderers/webgl/WebGLBackground.js

```diff
@@ -116,7 +116,7 @@ function WebGLBackground( renderer, state, geometries, premultipliedAlpha ) {
 		setClearColor: function ( color, alpha ) {
 
 			clearColor.set( color );
-			clearAlpha = alpha !== undefined ? alpha : 1;
+			clearAlpha = alpha !== undefined ? alpha : clearAlpha;
```

Comment:
... and consequently, this change would not be advisable...

Perhaps, we can remove the method `setClearAlpha()`, since `clearAlpha` can be set via `setClearColor()`.

That should eliminate some of the confusion.

Reply:
I think this is a good idea. 👍